### DISCUSSION TITLE
NGINX Unit

### DISCUSSION THREAD

**Discussion by jfmatth** [piku/piku#327] [Ideas]
So UNIT has been getting lots of stars today on GH, not sure how new it is, but might be a simpler way to use NGINX?

https://unit.nginx.org/

Thoughts?

**Comment by rcarmo** [comment]
It's an evolution of their old commercial product, which I used for a while. That used to bundle a Lua runtime, but the key selling point was that you can reconfigure `nginx` on the fly via an API, which was very useful for large commercial web serving clusters (and as a commercial Kubernetes ingress). It doesn't handle process supervision, I think (so it can't replace `uwsgi`), but it isn't an easier way to use `nginx` for us - the way we do it (with distribution-shipped packages) means it's much simpler to install than an extra product.

**Reply by jfmatth** [reply]
Thanks for the reply and through response :)
